### DISCUSSION TITLE
Route options

### DISCUSSION THREAD

**Discussion by Esmolante** [organicmaps/organicmaps#2973] [Ideas]
And the last one (for now):
It could be interesting if OM show the two or three best routes, when the calculated time is similar (not offering absurd alternatives, as often happens with Google Maps), for the driver to decide.

After testing several browsers using OSM, I have concluded that OM is the best. Or, at least, the one that best suits my needs. So, there is a lot of good and little bad to say. I've been telling the latter, which is the interesting thing to improve, but also your work merites recognition: An elegant and functional program, with the necessary functions that simply work. That's the way it should always be. ;)

Thank you very much!!!

**Comment by GreenAloe** [comment]
Already exists issue https://github.com/organicmaps/organicmaps/issues/1183

**Comment by Esmolante** [comment]
Sorry, I didn't find it (and I did some search, I promise).
